VAMobile/src/store/api/demo/mocks/appointments.json

```diff
@@ -597,11 +597,11 @@
             },
             "minutesDuration": 60,
             "phoneOnly": false,
-            "startDateLocal": "2022-06-02T20:00:00.000-04:00",
-            "startDateUtc": "2022-06-03T00:00:00.000Z",
+            "startDateLocal": "{{now + 45 minutes}}",
```

Comment:
Made this appointment 45 minutes in the future because we'll need it to test the check-in feature

Reply:
I'd make this like 47 minutes so we can see the state before it is check-in-able.